app/lib/features/pins/pages/pins_page.dart

```diff
@@ -43,9 +43,13 @@ class PinsPage extends ConsumerWidget {
                 ),
               ),
             ],
-            expandedContent: const Text(
-              'Pinned items from all the Spaces you are part of',
-            ),
+            expandedContent: size.width <= 600
+                ? null
+                : Text(
```

Comment:
I wonder if we can't just outsource that into a specific widget rather than copying that `<= 600` everywhere (also... why that number and why is it based on the `width`?)?

Reply:
We probably need to re-think about responsive architecture here. Maybe having some 3rd party package dealing with responsiveness or making a whole upper layer class wrapping all of it.